README.md

```diff
@@ -61,12 +72,12 @@ What are the values of these expressions?  Explain your answers.
 //We'll learn about require later in the course
 const ask = require('../lib/ask.js');
 
-let answer = 'not empty';
+let answer = x;
 
-while (answer !== '') {
-  answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
-   console.log("You guessed it!");
+while  {
+  x = ask
+  if (x !=> 1; x !<= 10; x++) {
+   console.log("You need to submit a number between 1 and 10");
```

Comment:
A good answer to this is

var answer = 'not empty';

while (answer !== '') {
    answer = ask("Guess my secret? ");
    if (answer >=1 && answer <=10) {
      console.log("You guessed it!");
      break;
   }else{
     console.log('Please try a number between 1 and 10')
 }
}